serverless-editor-demo.html

```diff
@@ -1,417 +1,429 @@
-  <script src='https://plugins.chamaileon.io/static/chamaileonSdk.js'></script>
-  <script>
-	new Vue({
-		el: '#app',
-	  	vuetify: new Vuetify(),
-	  	data() {
-			return {
-				chamaileonPlugins: null
-			}
-	  	},
-	  	async mounted() {
-		  	this.chamaileonPlugins = await chamaileonSdk.init({
-				mode: 'serverless',
-				whitelabel: {
-					urls: {
-						splashScreen: 'https://example.com',
-						createLogoJS: 'https://demo-staging.chamaileon.io/createLogo.js',
-					},
-					colors: {
-						'primary': '#FF0000',
-						'secondary': '#009f4a',
-						'red': '#ff5546',
-						'darkBlue': '#2d3291',
-						'darkGreen': '#00af6e',
-						'lightGreen': '#50d791',
-						'weirdGreen': '#50d791',
-						'pink': '#ff91a0',
-						'yellow': '#ffd23c',
-					},
-				},
-			})
-		},
-		methods: {
-			async openEditor() {
-				const self = this
-
-				let emailJsonToLoad = { // starting with an empty email
-					'eid' : 'root',
-					'type' : 'body',
-					'children' : [],
-					'attrs' : {
-						'usedColors' : [
-							'#595959',
-							'#000000',
-							'#cccccc',
-							'#cc0000',
-							'#FCFCFC',
-							'#5555ff',
-							'#FDFDFD',
-							'#FBFBFB',
-							'#999999',
-							'#f2f2f2',
-							'#ffffff'
-						]
-					},
-					'style' : {
-						'background-color' : '#FFFFFF',
-						'background-position' : 'center center',
-						'background-repeat' : 'no-repeat'
-					},
-					'version' : '2.0.0'
-				}
-
-				const userData = {
-					name: 'John',
-					avatar: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRgVUEjaWnHvhNaEy1-Jl6Ljvi7ahounqegSQ',
-				}
-				const blockArray = {
-					'email-blocks': [],
-					'john-doe-favs': [],
-				}
-
-				async function onSave({ emailJson }) {
-					return new Promise(async function (resolve) {
-						console.log('Email saved. emailJson:', emailJson)
-						return resolve()
-					})
-				}
-
-				async function onAutoSave({ emailJson }) {
-					return new Promise(async function (resolve) {
-						console.log('onAutoSave called. emailJson', emailJson)
-						return resolve()
-					})
-				}
 
-				async function onChange() {
-					return new Promise(function (resolve) {
-						console.log('onChange hook called')
-						return resolve()
-					})
-				}
-
-				async function onBeforeClose() {
-					return new Promise(async resolve => {
-						setTimeout(resolve, 1000) // simulating async
-					})
-				}
-
-				async function onAfterClose() {
-					return new Promise(function (resolve) {
-						console.log('Editor has been closed.')
-						return resolve()
-					})
-				}
-
-				async function onEditTitle({ title }) {
-					return new Promise(function (resolve) {
-						console.log('Email title has been updated to: ', title)
-						setTimeout(resolve, 1000) // simulating async
-					})
+<body>
+	<div id='app'>
+		<v-app>
+			<v-main>
+				<v-container>
+					<v-text-field
+						v-model="apiKey"
```

Comment:
Please do not mix double and single quotes. We use single in the example script.